### PR TITLE
Support S01 Close Out

### PR DESCRIPTION
**Pull request by DanielMwendwa**

### CODE REVIEW

File: configurations/kalobeyei_pipeline_config.json [hunk 1]
```diff
@@ -94,6 +95,10 @@
     {"RapidProKey": "Impact_Made (Time) - wusc_leap_s01_kalobeyei_programme_evaluation", "PipelineKey": "sent_on"},
     {"RapidProKey": "Impact_Made (Run ID) - wusc_leap_s01_kalobeyei_programme_evaluation", "PipelineKey": "impact_made_run_id"},
 
+    {"RapidProKey": "S01_Close_Out (Text) - wusc_leap_s01_kalobeyei_close_out_activation", "PipelineKey": "s01_close_out_raw", "IsActivationMessage": true},
+    {"RapidProKey": "S01_Close_Out (Time) - wusc_leap_s01_kalobeyei_close_out_activation", "PipelineKey": "sent_on"},
+    {"RapidProKey": "S01_Close_Out (Run ID) - wusc_leap_s01_kalobeyei_close_out_activation", "PipelineKey": "close_out_run_id"},
```
Comment (IsaackMwenda) [inline configurations/kalobeyei_pipeline_config.json:100]:
```suggestion
    {"RapidProKey": "S01_Close_Out (Run ID) - wusc_leap_s01_kalobeyei_close_out_activation", "PipelineKey": "s01_close_out_run_id"},
```

File: configurations/coding_plans.py [hunk 2]
```diff
@@ -183,6 +183,26 @@ def clean_age_with_range_filter(text):
                ],
                ws_code=CodeSchemes.KAKUMA_WS_CORRECT_DATASET.get_code_with_match_value(
                    "s01 leap kalobeyei impact made"),
+               raw_field_fold_strategy=FoldStrategies.concatenate),
+
+    CodingPlan(raw_field="s01_close_out_raw",
+               dataset_name="s01_close_out",
+               time_field="sent_on",
+               run_id_field="close_out_run_id",
```
Comment (IsaackMwenda) [inline configurations/coding_plans.py:191]:
```suggestion
               run_id_field="s01_close_out_run_id",
```

File: code_schemes/kalobeyei_s01_close_out.json [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+            "CodeID": "code-S-461bfcdf",
+            "CodeType": "Meta",
+            "MetaCode": "statement",
+            "DisplayText": "meta: statement",
+            "NumericValue": -100090,
+            "StringValue": "statement",
+            "VisibleInCoda": true,
+            "Shortcut": "s"
+        },
+        {
```
Comment (IsaackMwenda) [inline code_schemes/kalobeyei_s01_close_out.json:45]: use the code scheme structure for leap and no the one in oxfam

**Review by IsaackMwenda — APPROVED** [review]

File: code_schemes/kalobeyei_s01_close_out.json (hunk 3 above)
Comment (DanielMwendwa) [reply code_schemes/kalobeyei_s01_close_out.json]: updated thanks.

**Pull request merged**